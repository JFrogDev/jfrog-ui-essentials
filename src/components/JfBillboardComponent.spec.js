import {shallowMount} from '@vue/test-utils';
import JfBillboardComponent from './JfBillboardComponent.vue';

describe('JfBillboardComponent', () => {

    it('Contains an element to render graph', () => {
        const wrapper = shallowMount(JfBillboardComponent, {
            propsData: {}
        });
        expect(wrapper.find({ ref: 'chart' }).exists()).toBe(true);
    });
    
    it('contains a prop options', () => {
        const wrapper = shallowMount(JfBillboardComponent, {
            propsData: {
                options: 'chart-data'
            }
        });
        expect(wrapper.props().options).toBe('chart-data');
    });

});
