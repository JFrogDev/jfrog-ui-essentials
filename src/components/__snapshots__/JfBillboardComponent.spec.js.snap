// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JfBillboardComponent should match the snapshot 1`] = `<div />`;
